test/db/tools/rasm2

```diff
@@ -600,3 +600,30 @@ EXPECT=<<EOF
 mr r10, r4
 EOF
 RUN
+
+NAME=rasm2 -a arm.as -b 16 "wfe"
```

Comment:
Tests should require external stuff to be installed i dont expect the ci to have binutils for arm, ppc, mips..

Maybe those tests would require a condition or add those toolchains

Reply:
They are (and have to be due to lack of binutils on win/mac/s390x ...) marked broken. I've added needed packages to arm64 and ppc on travis.